supertokens_python/framework/fastapi/fastapi_request.py

```diff
@@ -60,7 +60,9 @@ def set_session_as_none(self):
         self.request.state.supertokens = None
 
     def get_path(self) -> str:
-        return self.request.url.path
+        root_path = self.request.scope.get("root_path")
+        url = self.request.url.path
+        return url[url.startswith(root_path) and len(root_path) :]
```

Comment:
- add tests for this?
- add comments exaplaining this? 
- What are all the edge cases this covers?